pkgs/development/tools/misc/doclifter/default.nix

```diff
@@ -1,14 +1,33 @@
-{ lib, stdenv, fetchurl, python3, makeWrapper, libxml2 }:
+{
+  fetchurl,
+  lib,
+  libxml2,
+  makeWrapper,
+  python3,
+  stdenv,
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "doclifter";
   version = "2.21";
+
   src = fetchurl {
-    url = "http://www.catb.org/~esr/${pname}/${pname}-${version}.tar.gz";
-    sha256 = "sha256-3zb+H/rRmU87LWh0+kQtiRMZ4JwJ3tVrt8vQ/EeKx8Q=";
+    url = "http://www.catb.org/~esr/doclifter/doclifter-${finalAttrs.version}.tar.gz";
+    hash = "sha256-3zb+H/rRmU87LWh0+kQtiRMZ4JwJ3tVrt8vQ/EeKx8Q=";
   };
+
+  postPatch = ''
+    substituteInPlace manlifter \
+      --replace-fail '/usr/bin/env python2' '/usr/bin/env python3'
```

Comment:
I assume this is not a shebang?

Reply:
Using `${python3.interpreter}` is not applicable? We don't have /usr/bin/env in our nix sandbox.

Reply:
It is a shebang and will be further patched during fixup.